recursive-list-main/src/pages/home/home.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from "@testing-library/react";
+import { ListData } from "components";
+import { MemoryRouter, Route, Routes } from "react-router-dom";
+import HomePage from "./home.component";
+
+describe("Home Page", () => {
+  it("Home Page is rendered properly", async () => {
+    render(
+      <MemoryRouter>
+        <HomePage />
+      </MemoryRouter>
+    );
+
+    expect(screen.getByTestId("home_svg")).toHaveAttribute("href", "/");
+  });
+});
+
+it("List data component is rendered properly", async () => {
```

Comment:
i think when you write test case with "it", the sentence should start from "Should". Pls check
```suggestion
it("Should render the list data component", async () => {
```

Reply:
Made necessary changes